dicomanonymizer/simpledicomanonymizer.py

```diff
@@ -144,7 +144,7 @@ def delete_element(dataset, element):
     """
     if element.VR == 'DA':
         replace_element_date(element)
```

Comment:
this always puzzles me: how a "delete" function does not delete but "replace" a date :-/